patches/MC-35514__fix_shipping_label_creation_issue__2.4.0.patch

```diff
@@ -0,0 +1,244 @@
+diff --git a/vendor/magento/module-shipping/Block/Adminhtml/Order/Packaging.php b/vendor/magento/module-shipping/Block/Adminhtml/Order/Packaging.php
+index e5e419328eea..827b26c64b33 100644
```

Comment:
Please remove this line as it is redundant.

Reply:
Fixed.